index.html

```diff
@@ -234,18 +236,19 @@ <h2>I'm always interested in hearing about new projects, so if you'd like to cha
         <footer>
             <ul class="network">
                 <li class="social">
-                    <a target="_blank" href="https://github.com/Kakalanp" rel="noopener"><img src="https://img.icons8.com/ios-glyphs/30/000000/github.png" alt="g"/></a>
+                    <a target="_blank" href="https://github.com/Kakalanp" rel="noopener noreferrer"><img src="https://img.icons8.com/ios-glyphs/30/000000/github.png" alt="GitHub profile"/></a>
                 </li>
                 <li class="social">
-                    <a target="_blank" href="https://www.linkedin.com/in/joaquin-garrido-lecca-zanetti-623583204/" rel="noopener"><img src="https://img.icons8.com/ios-glyphs/30/000000/linkedin-2--v1.png" alt="li"/></a>
+                    <a target="_blank" href="https://www.linkedin.com/in/joaquin-garrido-lecca-zanetti-623583204/" rel="noopener noreferrer"><img src="https://img.icons8.com/ios-glyphs/30/000000/linkedin-2--v1.png" alt="Linkedin profile"/></a>
                 </li>
                 <li class="social">
-                    <a target="_blank" href="https://www.twitter.com/LeccaJoaquin"  rel="noopener"><img src="https://img.icons8.com/ios-glyphs/30/000000/twitter--v1.png" alt="t"/></a>
+                    <a target="_blank" href="https://www.twitter.com/LeccaJoaquin"  rel="noopener noreferrer"><img src="https://img.icons8.com/ios-glyphs/30/000000/twitter--v1.png" alt="Twitter profile"/></a>
                 </li>
```

Comment:
[Required]
- [ ] Same like for all other changes - please make sure that you added links to your changes into your report as it was required.

[Optional] 

- [ ] You can revert those commits and make changes on a new branch. Then you can add link to changes from that branch to your report on this branch.